config/fxdata/magic.cfg

```diff
@@ -18,28 +18,69 @@
 
 [spell0]
 Name = NOSPELL
-; Whether the spell can target a thing or not.
+; Determines whether the spell can be cast on the caster or on a target. 0: caster, 1: target. The caster can also be the target.
 CastAtThing = 0
+; The [shot] the spell is taking the stats from.
 ShotModel = NOSHOT
 ; Effect or 'effectElement' (negative numbers) that is created as the spell. Accepts both names and numbers.
 EffectModel = NULL
 ; The [power] the spell is taking the stats from.
 SpellPower = NOPOWER
-; Spell effects the target will be affected by. Powers of 2 for unique effects, add them up to combine.
+; Spell effects the target will be affected by. Accepts both names and numbers. Powers of 2 for unique effects, add them up to combine.
+;   SLOW - Halves movement speed and doubles instances cooldowns. 'SPELL_SLOW' by default.
```

Comment:
Not sure what value the `'SPELL_SLOW' by default.` has here. I think it would confuse more people than it would help.